resources/migrations/035_modify_setting_value_length.yaml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+  - changeSet:
+      id: 35
+      author: agilliland
+      changes:
+        - modifyDataType:
+            tableName: setting
+            columnName: value
+            newDataType: TEXT
```

Comment:
yeah.  i saw that too.  oddly it works without this against MySQL which is what i was testing against.